en-US/api/leaderboard.md

```diff
@@ -2,29 +2,35 @@
 name: Leaderboard
 ---
 
-- [GET `/v1/leaderboard`](#get-v1leaderboard)
-    - [Parameters](#parameters)
-    - [Example Response](#example-response)
-- [GET `/v1/leaderboard/hits`](#get-v1leaderboardhits)
-    - [Parameters](#parameters-1)
-    - [Example Response](#example-response-1)
+## Table of Contents
 
-## GET `/v1/leaderboard`
+- [Table of Contents](#table-of-contents)
+- [Endpoints](#endpoints)
+    - [GET `/leaderboard`](#get-%2Fleaderboard)
+        - [Parameters](#parameters)
+        - [Example Response](#example-response)
+    - [GET `/leaderboard/hits`](#get-%2Fleaderboard%2Fhits)
+        - [Parameters](#parameters-1)
+        - [Example Response](#example-response-1)
+
+## Endpoints
+
+### GET `/leaderboard`
 
 Retrieves the top 50 users on the leaderboard
 
-### Parameters
+#### Parameters
 
 | Name    | Description       | Required |
 | ------- | ----------------- | -------- |
 | mode    | Game mode         | Yes      |
 | country | Filter by country | No       |
 | page    | Pagination        | No       |
 
-### Example Response
+#### Example Response
 
 ```json
-// https://api.quavergame.com/v1/leaderboard?mode=2&country=us
+// https://api.quavergame.com/leaderboard?mode=2&country=us
```

Comment:
This link change is incorrect, and the same issue applies elsewhere in the same commit.